### DISCUSSION TITLE
关于 Form 中组件激活热区的疑惑

### DISCUSSION THREAD

**Discussion by NNodkkd** [DouyinFE/semi-design#2116] [General]
为什么 Form 中 Input 的 label 是 Input 的激活热区、select 的 label 不是？
主要是疑惑为什么 label 要作为 input 的激活热区。
<img width="736" alt="企业微信截图_1a1b25e1-8750-4dd9-b2c2-8fc1131c17e6" src="https://github.com/DouyinFE/semi-design/assets/112705506/5cd4d8bd-75c9-4a71-8fc5-ae4dda9690fc">

**Comment by pointhalo** [comment]
html的label标准行为，https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/label
```
<div class="preference">
  <label for="cheese">Do you like cheese?</label>
  <input  name="cheese" id="cheese" />
</div>
```

如果组件是沿用原生 html element 渲染的，那就会有这个行为。主要是 label for 和 id 的组合作用。
像 Select 这种自渲染的element，则不会有这种行为。

**Reply by NNodkkd** [reply]
团队内的交互设计师有讨论过在用户视角这个标准行为是否合理吗

**Reply by pointhalo** [reply]
这是HTML标准。

**Reply by pointhalo** [reply]
遵循A11y 规范是需要这么实现的。而不是自己用div 之类的替换 label。
